### PR TITLE
adding Inferential Stats baseline linear regression model

### PR DESCRIPTION
**Pull request by nishalpattan**
Hi @rcuevass , Added Inferential Stats & Baseline linear regression model. Please review .

-Nishal

### CODE REVIEW

File: Capstone_Project/BaseLine.ipynb [hunk 1]
```diff
@@ -1,5783 +0,0 @@
...
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 227,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# have a single column of visitors instead of two\n",
-    "df_total['visitors'] = df_total['visitors'] + df_total['reserve_visitors']"
```
Comment (rcuevass) [inline Capstone_Project/BaseLine.ipynb:831]: @nishalpattan :
Why adding visitors plus reserve visitors? Can you give me some context so that I can understand?

File: Capstone_Project/BaseLine.ipynb [hunk 2]
```diff
@@ -1,5783 +0,0 @@
...
-    "#Add new features fri/Sat/Sun\n",
-    "#fri == 4; sat == 5; sun == 6\n",
-    "df_total['Week_day'] = df_total['visit_date'].dt.dayofweek\n",
-    "df_total['day'] = df_total['visit_date'].dt.day\n",
-    "df_total['month'] = df_total['visit_date'].dt.month\n",
-    "df_total['year'] = df_total['visit_date'].dt.year\n",
-    "df_total['fri'] = (df_total['Week_day'] == 4)*1\n",
-    "df_total['sat'] = (df_total['Week_day'] == 5)*1\n",
-    "df_total['sun'] = (df_total['Week_day'] == 6)*1\n",
-    "df_total['geo_location'] = (df_total['latitude'] + df_total['longitude'])\n",
```
Comment (rcuevass) [inline Capstone_Project/BaseLine.ipynb:1323]: @nishalpattan : why adding latitude plus longitude? Why not considering the coordinates directly? that is (x,y)

File: Capstone_Project/BaseLine.ipynb [hunk 3]
```diff
@@ -1,5783 +0,0 @@
...
-    }
-   ],
-   "source": [
-    "df_total.head()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 237,
-   "m
```
Comment (rcuevass) [inline Capstone_Project/BaseLine.ipynb:2207]: @nishalpattan : I wonder if it would be better to show a possible relationship differently. For instance, I am thinking an XY plot so that at each (x,y) you show a circle whose size represents the number of customers associated with the restaurant in that coordinate? How hard/easy would it be to make a plot of that sort?

Comment (rcuevass) [inline Capstone_Project/BaseLine.ipynb:2275]: @nishalpattan : quick change in typo: golidays ==> holidays

Comment (rcuevass) [inline Capstone_Project/BaseLine.ipynb:2969]: @nishalpattan : It would be nice to do a split so that you have training/validation/test sets. What do you think?

Comment (rcuevass) [inline Capstone_Project/BaseLine.ipynb:2985]: @nishalpattan : what is the RMSE for the test set?

Comment (rcuevass) [reply Capstone_Project/BaseLine.ipynb:2985]: @nishalpattan : Perhaps not for this PR and more for you to check later but it would be worth for you to check concepts on k-fold cross-validation. I would suggest to do some googling but perhaps you could start with :
- https://en.wikipedia.org/wiki/Cross-validation_(statistics)
- http://scikit-learn.org/stable/modules/cross_validation.html

Comment (rcuevass) [reply Capstone_Project/BaseLine.ipynb:2985]: @nishalpattan : I think that a visually informative way of showing the performance of the mode would be to plot predicted vs. actual values in a scatter plot. What do you think?

File: Capstone_Project/BaseLine.ipynb (hunk 1 above)
Comment (nishalpattan) [reply Capstone_Project/BaseLine.ipynb:831]: I thought both would represent the single feature, like the total number of visitors.

File: Capstone_Project/BaseLine.ipynb (hunk 2 above)
Comment (nishalpattan) [reply Capstone_Project/BaseLine.ipynb:1323]: I am doubted about this feature, that both longitude and latitude contribute as a single feature as geo location.

Comment (nishalpattan) [reply Capstone_Project/BaseLine.ipynb:1323]: I might be wrong in doing this.

File: Capstone_Project/BaseLine.ipynb (hunk 1 above)
Comment (rcuevass) [reply Capstone_Project/BaseLine.ipynb:831]: @nishalpattan : Wild guess here but would not "reserve visitors" be the initial number of customers who in principle that were going to show at the restaurant but at the end only the number under "visitors" show?

File: Capstone_Project/Inferential_Statistics.ipynb [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+   "source": [
+    "p_Val = 1 - stats.t.cdf(t,df=df)\n",
+    "print(\"P-Value : \",p_Val)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<b> Here the p-Value is less than than the significant level $\\alpha$, so Rejecting the null hypothesis $H_{0}$ the mean vistor count for izakaya and other genres are same </b>"
```
Comment (rcuevass) [inline Capstone_Project/Inferential_Statistics.ipynb:322]: @nishalpattan : good idea in making this hypothesis test. Could I suggest to rephrase the result of the test in more "plain english"? For instance: "there is (or there is not, according to the result) statistical evidence that the mean of ..... "
What do you think?

**Comment by rcuevass** [comment]
@nishalpattan : 
Added some comments and suggested tasks. Let me know what you think.
Good job on this; getting there!

File: Capstone_Project/BaseLine.ipynb (hunk 1 above)
Comment (nishalpattan) [reply Capstone_Project/BaseLine.ipynb:831]: yes rogelio, i will revert this and add reserve_visitors as single feature, but i am having more null(NaN) values in reserve visitors feature, can I replace them with 0 or any alternate way of handling them.
